tools/dep_updaters/update-base64.sh

```diff
@@ -51,6 +51,10 @@ mv "$WORKSPACE/base64" "$DEPS_DIR/base64/"
 # to work, we create it and leave it empty.
 echo "// Intentionally empty" >> "$DEPS_DIR/base64/base64/lib/config.h"
 
+# Clear out .gitconfig, otherwise config.h is ignored. That's dangerous when
+# people check in our tarballs into source control and run `git clean`.
+echo "# Intentionally empty" >> "$DEPS_DIR/base64/base64/.gitconfig"
```

Comment:
```suggestion
echo "# Intentionally empty" >> "$DEPS_DIR/base64/base64/.gitignore"
```

Reply:
Grr, yes. That's what I get for making last-minute changes right before stepping out.